index.js

```diff
@@ -1,6 +1,10 @@
 'use strict';
 const electron = require('electron');
 
+if (typeof electron === 'string') {
```

Comment:
Why would it be a `string`? I think a better check would be
```suggestion
if (typeof electron !== 'object') {
```

Reply:
It would be a string when importing Electron in a non-Electron environment. See [Electron programmatic usage](https://www.npmjs.com/package/electron#programmatic-usage).